nearlib/nearclient.js

```diff
@@ -47,6 +48,21 @@ class NearClient {
     async request (methodName, params) {
         return await this.nearConnection.request(methodName, params);
     };
+
+    async generateNewKeyFromRandomSeed () {
+        var randomSeed = crypto.randomBytes(32);
+        var newKeypair = ed25519.MakeKeypair(randomSeed);
+        const response = {};
+        response['public_key'] = this.encodeBufferInBs58(newKeypair.publicKey);
+        response['secret_key'] = this.encodeBufferInBs58(newKeypair.privateKey);
+        return response;
```

Comment:
```js
return {
    public_key: this.encodeBufferInBs58(newKeypair.publicKey),
    secret_key: this.encodeBufferInBs58(newKeypair.privateKey)
} 
```

Reply:
same applies to other cases. Very preferable to construct object literal vs assign properties one by one.